build/images/perftool/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM ubuntu:18.04
+
+LABEL maintainer="Antrea <projectantrea-dev@googlegroups.com>"
+LABEL description="A Docker image based on Ubuntu 18.04 which is used for performance tests."
+
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends apache2-utils iperf3 && \
+    rm -rf /var/cache/apt/* /var/lib/apt/lists/*
+ENTRYPOINT "iperf3" "-s"
```

Comment:
This entrypoint seems a bit arbitrary to me. Why should the default behavior be to start an iperf server?

Reply:
This would reduce operations of test, no need to have an extra goroutine to execute the starting server command. This entry point would also make the pod created keeping running by default.